dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/XtxtUML.xtext

```diff
@@ -212,10 +212,18 @@ TUMultiplicity :
 
 // Commons
 
+TUModifiers:
+	{TUModifiers} (static?='static'? & visibility=TUVisibility? & externality=TUExternality?)
```

Comment:
I see now, it is required because all features of this rule are optional and during parsing, Xtext instantiates AST nodes only at the first actual assignment of a feature of the corresponding rule.